### PR TITLE
fix(delete): #DRIV-60 add feedback message and update typo when confirming the deletion

### PR DESCRIPTION
**Pull request by Matiyeu**
## Describe your changes

- Updated description text in delete confirmation modal
- Added a feedback message when deleting folder/document

## Checklist tests

- [x] Correct display of the notification on desktop/mobile
- [x] Correct display of the modal description 

## Issue ticket number and link

[DRIV-60](https://jira.support-ent.fr/browse/DRIV-60)

## Checklist before requesting a review (magic string, indentation, comment/documentation...)

- [x] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

**Review by sjollois — APPROVED** [review]

File: src/main/resources/i18n/en.json [hunk 1]
```diff
@@ -9,5 +9,7 @@
   "workspace.folder.create": "Create",
   "cancel": "Cancel",
   "folder.new": "Create folder",
-  "nextcloud.url.connect": "To connect from your phone or computer"
+  "nextcloud.url.connect": "To connect from your phone or computer",
+  "nextcloud.documents.confirm.remove": "Do you want to permanently delete these documents?",
```
Comment (qpic-cgi) [inline src/main/resources/i18n/en.json:13]: Tu peux voir avec ton techlead => Je ne sais pas si ça vaut le coup d'utiliser un html entity pour le "?" ou @JinDouang

File: src/main/resources/i18n/en.json [hunk 2]
```diff
@@ -9,5 +9,7 @@
...
+  "nextcloud.documents.toast": "Items have been permanently deleted."
```
Comment (qpic-cgi) [inline src/main/resources/i18n/en.json:14]: Tu peux lui donner un nom un peu plus déterminant, ex:
"nextcloud.documents.deletion.confirmation"

**Review by qpic-cgi — CHANGES_REQUESTED** [review]

File: src/main/resources/i18n/en.json (hunk 1 above)
Comment (alicedraillard) [reply src/main/resources/i18n/en.json:13]: utiliser les html entities https://www.w3schools.com/html/html_entities.asp

File: src/main/resources/i18n/en.json (hunk 2 above)
Comment (Matiyeu) [reply src/main/resources/i18n/en.json:14]: Je t'avoue que j'ai reprise celui du workspace mais oui effectivement, je modifie ça.

File: src/main/resources/i18n/en.json [hunk 3]
```diff
@@ -9,5 +9,7 @@
   "workspace.folder.create": "Create",
   "cancel": "Cancel",
   "folder.new": "Create folder",
-  "nextcloud.url.connect": "To connect from your phone or computer"
+  "nextcloud.url.connect": "To connect from your phone or computer",
+  "nextcloud.documents.deletion.confirmation": "Do you want to permanently delete these documents",
+  "nextcloud.documents.deletion.toast": "Items have been permanently deleted."
```
Comment (qpic-cgi) [inline src/main/resources/i18n/en.json:14]:
```suggestion
  "nextcloud.documents.confirm.deletion": "Do you want to permanently delete these documents",
  "nextcloud.documents.deletion.confirmation": "Items have been permanently deleted."
```

File: src/main/resources/public/template/behaviours/sniplet-nextcloud-content/toolbar/workspace-nextcloud-toolbar-delete.html [hunk 4]
```diff
@@ -7,7 +7,7 @@ <h2>
 
         <!-- Confirm deletion body text -->
         <div class="row">
-            <i18n>workspace.sniplet.documents.confirm.remove</i18n>
+            <i18n>nextcloud.documents.deletion.confirmation</i18n> &quest;
```
Comment (qpic-cgi) [inline src/main/resources/public/template/behaviours/sniplet-nextcloud-content/toolbar/workspace-nextcloud-toolbar-delete.html:10]:
```suggestion
            <i18n>nextcloud.documents.deletion.confirmation</i18n>&nbps;&quest;
```

**Review by alicedraillard — APPROVED** [review]

File: src/main/resources/public/ts/sniplets/content/workspace-nextcloud-toolbar.sniplet.ts [hunk 5]
```diff
@@ -125,6 +125,7 @@ export class ToolbarSnipletViewModel implements IViewModel {
         const paths: Array<string> = this.vm.selectedDocuments.map((selectedDocument: SyncDocument) => selectedDocument.path);
         this.vm.nextcloudService.deleteDocuments(model.me.userId, paths)
             .then(() => {
+                notify.info("nextcloud.documents.deletion.toast");
```
Comment (qpic-cgi) [inline src/main/resources/public/ts/sniplets/content/workspace-nextcloud-toolbar.sniplet.ts:128]:
```suggestion
                notify.info("nextcloud.documents.deletion.confirmation");
```

**Review by qpic-cgi — CHANGES_REQUESTED** [review]

**Review by qpic-cgi — APPROVED** [review]

**Pull request merged**